README.md

```diff
@@ -227,6 +227,15 @@ export class GqlThrottlerGuard extends ThrottlerGuard {
 }
 ```
 
+However, when using GraphQL, it's important to configure the context correctly in the GraphQLModule to avoid any problems. To ensure proper context setup, add the following line of code in your GraphQLModule configuration:
+
+```typescript
+GraphQLModule.forRoot({
+  // ... other GraphQL module options
+  context: ({ req, res }) => ({ req, res }),
```

Comment:
It should be noted this is the specific setup for Apollo Express, and that Apollo fastify, and mercurius have different contexts that need to be used

Reply:
You're right, I changed it again, now would it be ok or should we specify something else?